### PR TITLE
Minior updates on 'correcting' JSON to camelCase...

### PR DESCRIPTION
**Pull request by matzew**
Related to
* https://github.com/knative/eventing/pull/2373
* https://github.com/knative/eventing/pull/2400

## Proposed Changes

  * going with cameCase JSON

### CODE REVIEW

**Comment by vaikas** [comment]
I'd propose doing these in the v1beta1 as they are non-backwards compatible.

File: github/pkg/apis/sources/v1alpha1/githubsource_types.go [hunk 1]
```diff
@@ -77,7 +77,7 @@ type GitHubSourceSpec struct {
 
 	// API URL if using github enterprise (default https://api.github.com)
 	// +optional
-	GitHubAPIURL string `json:"githubAPIURL,omitempty"`
```
Comment (n3wscott) [inline github/pkg/apis/sources/v1alpha1/githubsource_types.go]: These will be breaking API changes.

**Comment by antoineco** [comment]
@matzew we can probably close this since we don't want to break APIs?

**Pull request closed without merging**